docs/en/operations/table_engines/mergetree.md

```diff
@@ -161,10 +164,33 @@ The number of columns in the primary key is not explicitly limited. Depending on
 
 - Provide additional logic when data parts merging in the [CollapsingMergeTree](collapsingmergetree.md#table_engine-collapsingmergetree) and [SummingMergeTree](summingmergetree.md#table_engine-summingmergetree) engines.
 
-    You may need many fields in the primary key even if they are not necessary for the previous steps.
+    If you need this, it makes sense to specify the *sorting key* that is distinct from the primary key.
 
 A long primary key will negatively affect the insert performance and memory consumption, but extra columns in the primary key do not affect ClickHouse performance during `SELECT` queries.
 
+<a name="table_engines-mergetree-sorting_key"></a>
+
+### Choosing the Sorting Key that is distinct from the Primary Key
```

Comment:
"distinct" might get confused with keyword, I'd just say "...Sorting Key that Differs from The Primary Key"